releases/v1.5.0-rc.toml

```diff
@@ -89,5 +89,44 @@ containers the same set of permissions as the calling process, preventing errors
 when privileged containers attempt to get more permissions than given to the
 caller.
 
+#### Configuration changes
+
+##### *New* `registry.config_path` for CRI plugin
+
+`registry.config_path` specifies a directory to look for registry hosts
+configuration. When resolving an image name during pull operations, the CRI
+plugin will look in the `<registry.config_path>/<image hostname>/` directory
+for host configuration. A `hosts.toml` file in that directory may be used
+to configure which hosts will be used for the pull operation as well
+host-specific configurations. Updates under that directory do not require
+restarting the containerd daemon.
+
+Enable `registry.config_path` in the containerd configuration file.
+
+```toml
+[plugins."io.containerd.grpc.v1.cri".registry]
+   config_path = "/etc/containerd/certs.d"
+```
+
+Configure registry hosts, such as `/etc/containerd/certs.d/docker.io/hosts.toml`
+for any image under the `docker.io` namespace (any image on Docker Hub).
+
+```
+server = "https://registry-1.docker.io"     # Exclude this to not use upstream
+
+[host."https://public-mirror.example.com"]
+  capabilities = ["pull"]                   # Requires less trust, won't resolve tag to digest from this host
+[host."https://docker-mirror.internal"]
+  capabilities = ["pull", "resolve"]
+  ca = "/etc/containerd/certs.d/docker.io/docker-mirror.crt"
```

Comment:
this is either a full abs path or relative to the same directory which the `hosts.toml` is in right?